### PR TITLE
CAMEL-20062: fix the split package warning in Camel Quarkus

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

**Comment by orpiske** [comment]
Waiting for a full test cycle on my own CI before marking this as ready to merge.

**Review by oscerd — APPROVED** [review]

**Review by davsclaus — APPROVED** [review]

**Comment by orpiske** [comment]
CI tests ran Ok. Merging this one ...

**Pull request merged**
